projects/ion/src/lib/tour/tour-backdrop/tour-backdrop.component.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { render, RenderResult, screen } from '@testing-library/angular';
+
+import { IonTourStepProps } from '../../core/types';
+import { IonTourBackdropComponent } from './tour-backdrop.component';
+
+const DEFAULT_PROPS: Partial<IonTourBackdropComponent> = {};
+
+const sut = async (
+  props: Partial<IonTourBackdropComponent> = {}
+): Promise<RenderResult<IonTourBackdropComponent>> => {
+  return render(IonTourBackdropComponent, {
+    declarations: [IonTourBackdropComponent],
+    componentProperties: {
+      ...DEFAULT_PROPS,
+      ...props,
+    },
+  });
+};
```

Comment:
DEFAULT_PROPS is empty and is only used in the SUT.

```suggestion
const sut = async (
  props: Partial<IonTourBackdropComponent> = {}
): Promise<RenderResult<IonTourBackdropComponent>> => {
  return render(IonTourBackdropComponent, {
    declarations: [IonTourBackdropComponent],
    componentProperties: {
      ...props,
    },
  });
};
```